### ISSUE TITLE
Feature request: Handle security account with multiple deposit accounts

### ISSUE THREAD

**Issue by lsolesen** [buchen/portfolio#1530]
It is common to have currency accounts attached to a security account. It would be very helpful if it was possible somehow to get a proper XIRR on a widget on the dashboard with a setup like this:

Security Account Nordnet
Deposit Account Nordnet USD
Deposit Account Nordnet EUR

Right now it seems that there is no way to create a group where you can get the XIRR for that entity.

See https://forum.portfolio-performance.info/t/showing-xirr-on-the-dashboard-for-a-multicurrency-deposit-account/8921

**Comment by chirlu** [comment]
Why can’t you create a filter for those accounts?

**Comment by lsolesen** [comment]
As far as I can tell filters are not available on the Performance Dashboard when adding a widget. I can group securities and deposit accounts using taxonomies, but I am pretty sure the XIRR is not calculated corretly when doing that. Also that makes it necesary to manually adding securites to the taxonomy, if I change the investments.

**Comment by lsolesen** [comment]
I did a couple of experiments. It seems that the XIRR comes closer to something correct, if I also added securities whichs has been sold from the account. It might be fixed by #1524 if it is also possible to put the security accounts (and with that all the securities which has been handled by the security account) into taxonomies. Then it would be possible handle this. Another option could be to make it possible to add more deposit accounts to a security account (however, that is probably a bigger change in the architecture of the program).
